docs/tutorials/proposals/new-market-proposal.md

```diff
@@ -97,6 +97,7 @@ An instrument contains the following properties:
 | `dataSourceSpecForSettlementData` | This defines the data source that will be used to identify the settlement price when the market expires. | prices.ORANGES.value |
 | `dataSourceSpecForTradingTermination` | The fields that define the data source used for terminating trading on the market. | vegaprotocol.builtin.timestamp |
 | `dataSourceSpecBinding` | The fields describe how specific information provided by the data source is used. For example, they can identify the specific name of the settlement price output, or the specific name of the trading termination property. |
+| `sourceChainId` | Describes the chain ID of the data source. This chain must already be enabled in network parameters and supported by validators. |
```

Comment:
I do not think it is required here. Here is the example proposal: https://github.com/vegaprotocol/testnet-proposals/blob/main/incentive-03-02-2024/market-btcusdt.json. the `sourceChainId` is part of the `dataSourceSpecForSettlementData` and the `dataSourcesSpec`

Reply:
@daniel1302 I can see how that's confusing! It's in the correct place in the actual template, this is more of a guideline of what each field means. But I can separate out the data source fields more clearly into a different table.